gossip/c_block_callbacks_test.go

```diff
@@ -864,6 +869,268 @@ func TestIsPermissible_DetectsNonPermissibleTransactions(t *testing.T) {
 	}
 }
 
+func TestRlpEncodedMaxHeaderSizeInBytes_IsAnUpperBound(t *testing.T) {
+	setToMax := func(size int) []byte {
+		b := bytes.Repeat([]byte{0xFF}, size)
+		return b
+	}
+
+	maxHash := common.Hash(setToMax(32))
+	maxAddress := common.Address(setToMax(20))
+	maxBloom := types.Bloom(setToMax(2048))
+	maxBlock := types.BlockNonce(setToMax(8))
+	maxUint64 := uint64(math.MaxUint64)
+
+	// Geth sanity check for extra defines an upper bound of 100 * 1024 bytes.
```

Comment:
this line is a bit confusing for me. `geth sanity check*s* for extra defines` or `geth sanity check defines an extra upper bound` ?